<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiimotedevShortcutsConfig</class>
 <widget class="QWidget" name="WiimotedevShortcutsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Shortcuts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/data.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="wiimotedev_enable">
     <property name="text">
      <string>Enable Wii remote support in Clementine</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="wiimotedev_groupbox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="layout">
      <item>
       <widget class="QGroupBox" name="wiimotedev_groupbox">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="wiimotedev_active">
           <property name="text">
            <string>Connect Wii remotes to Clementine using active/deactive action</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="wiimotedev_focus">
           <property name="text">
            <string>Enable shortcuts only when application is focused</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="wiimotedev_notification">
           <property name="text">
            <string>Use notifications to report Wii remote status</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Use Wii remote</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="wiimotedev_device">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="list">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Buttons</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Action</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="wiimotedev_add_action">
             <property name="text">
              <string>Add action</string>
             </property>
             <property name="icon">
              <iconset resource="../../data/data.qrc">
               <normaloff>:/icons/22x22/list-add.png</normaloff>:/icons/22x22/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="wiimotedev_delete_action">
             <property name="text">
              <string>Remove action</string>
             </property>
             <property name="icon">
              <iconset resource="../../data/data.qrc">
               <normaloff>:/icons/22x22/list-remove.png</normaloff>:/icons/22x22/list-remove.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="wiimotedev_reload">
             <property name="text">
              <string>Defaults</string>
             </property>
             <property name="icon">
              <iconset resource="../../data/data.qrc">
               <normaloff>:/icons/22x22/view-refresh.png</normaloff>:/icons/22x22/view-refresh.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../data/data.qrc"/>
 </resources>
 <connections/>
</ui>
